projects/scion/microfrontend-platform/src/lib/safe-runner.ts

```diff
@@ -21,5 +21,6 @@ export function runSafe<T = void>(runnable: () => T): T {
   }
   catch (error) {
     Beans.get(Logger).error('[UnexpectedError] An unexpected error occurred.', error);
+    throw error;
```

Comment:
It is a safe runner intended to swallow errors. Instead, return `undefined`.

Reply:
done